teps/0084-endtoend-provenance-collection.md

```diff
@@ -212,6 +394,31 @@ Go in to as much detail as necessary here.
 This might be a good place to talk about core concepts and how they relate.
 -->
 
+<!-- TODO: Revisit this once event-payload is taken into consideration -->
+
+**Multiple Attestations**: With this proposal, an image may have multiple attestations, one for the
+taskrun and one for the pipelinerun. When using the OCI storage, for example, the attestation image
+will contain 2 layers each representing one of the attestations. This is not necessarily a new
+behavior as the existing mechanism to attest taskruns could also produce multiple attestations for
+a single image under certain circumstances. However, this is much more likely to happen with this
+proposal.
+
+**No Signature from PipelineRun**: A pipelinerun that produces the expected type hinting result
+will cause a pipelinerun attestation to be created. However, it will not cause an image signature
+to also be created. The type hinting result on the taskrun remains the mechanism for signing
+images. This proposal relies on using the same signing key for processing both taskruns and
+pipelineruns, thus making multiple signatures redundant. This may have to be revisited in the
+future if taskrun attestations become deprecated/optional, or support for different signing keys
+for different artifact types is introduced.
```

Comment:
> This proposal relies on using the same signing key for processing both taskruns and
pipelineruns, thus making multiple signatures redundant.

By same signing key, do you mean pipeline level type hinting result will be the same as taskrun level i.e. "IMAGE_URL", and "IMAGE_DIGEST"?